common/src/main/templates/io/netty/util/collection/KObjectMap.template

```diff
@@ -76,35 +71,14 @@ public interface @K@ObjectMap<V> {
     V remove(@k@ key);
 
     /**
-     * Returns the number of entries contained in this map.
-     */
-    int size();
-
-    /**
-     * Indicates whether or not this map is empty (i.e {@link #size()} == {@code 0]).
-
-     */
-    boolean isEmpty();
-
-    /**
-     * Clears all entries from this map.
-     */
-    void clear();
-
-    /**
      * Indicates whether or not this map contains a value for the specified key.
      */
     boolean containsKey(@k@ key);
 
     /**
-     * Indicates whether or not the map contains the specified value.
-     */
-    boolean containsValue(V value);
-
-    /**
      * Gets an iterable collection of the entries contained in this map.
      */
-    Iterable<Entry<V>> entries();
+    Iterable<PEntry<V>> entries();
```

Comment:
@nmittler alternatively have the primitive map `implement Iterable<PEntry<V>>` and make this an Iterator?